src/main/scala/cromwell/util/google/GoogleScopes.scala

```diff
@@ -1,12 +1,17 @@
 package cromwell.util.google
 
+import com.google.api.services.genomics.GenomicsScopes
 import com.google.api.services.storage.StorageScopes
+import scala.collection.JavaConverters._
 
 object GoogleScopes {
-  val Scopes = Vector(
-    StorageScopes.DEVSTORAGE_FULL_CONTROL,
-    StorageScopes.DEVSTORAGE_READ_WRITE,
-    "https://www.googleapis.com/auth/genomics",
-    "https://www.googleapis.com/auth/compute"
-  )
+  val Scopes = {
+    val scopes = Vector(
+      StorageScopes.DEVSTORAGE_FULL_CONTROL,
+      StorageScopes.DEVSTORAGE_READ_WRITE,
+      "https://www.googleapis.com/auth/genomics",
+      "https://www.googleapis.com/auth/compute"
+    ) ++ GenomicsScopes.all().asScala
+    scopes.sorted.distinct
```

Comment:
No mention of scopes requiring ordering in the docs. All Google API code seems to use `Collection`, not `List`.

* https://developers.google.com/identity/protocols/OAuth2ServiceAccount
* https://github.com/google/google-api-java-client/blob/1.20.0/google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/GoogleCredential.java#L463-L477
* https://github.com/google/google-oauth-java-client/blob/1.20.0/google-oauth-client/src/main/java/com/google/api/client/auth/oauth2/TokenRequest.java#L201-L204